llumnix/entrypoints/vllm/api_server.py

```diff
@@ -224,35 +224,44 @@ async def is_ready():
     ready_status = await engine_manager.is_ready.remote()
     return ready_status
 
+class CustomArgumentParser(argparse.ArgumentParser):
```

Comment:
ArgumentParserLlumnix? which is more consistent with our naming style.

Reply:
get